crates/rust-analyzer/src/global_state.rs

```diff
@@ -79,7 +82,9 @@ pub(crate) struct GlobalState {
     pub(crate) source_root_config: SourceRootConfig,
     pub(crate) proc_macro_client: Option<ProcMacroClient>,
     pub(crate) workspaces: Arc<Vec<ProjectWorkspace>>,
-    pub(crate) fetch_workspaces_queue: OpQueue,
+    pub(crate) fetch_workspaces_queue: OpQueue<Arc<Config>>,
```

Comment:
Hm, I'd expect `OpQueue<()>` here.

```suggestion
    pub(crate) fetch_workspaces_queue: OpQueue<Arc<Config>>,
```

But I guess `Arc<Config>` would be semantically more appropriate.

Reply:
I changed to () :)